### ISSUE TITLE
Chrono Trigger (EU) - Crash in menu

### ISSUE THREAD

**Issue by mrRosset** [mgba-emu/mgba#708]
Tested on medusa alpha 2 and medusa nightly 2017-04-26 commit f32fbd737fee9defcfbbb73a386685f936eeaae5

**OS:** Windows 10 Edu x64
**CPU:** i7-3540M
**GPU:** intel hd 4000

Chrono Trigger crashes when pressing "New Game" after finishing the intro cinematic.

![Crash gif](http://i.imgur.com/NX7Dn4K.gif)

**Comment by Anty-Lemon** [comment]
Confirmed on Debian.

Build: 0.1-medusa-4895-f32fbd73
OS: Debian 9.0 stretch amd64
Game: Chrono Trigger US (YQUEGD)

I'm pretty sure this is a regression. Tracking down which commit broke it...

**Comment by Anty-Lemon** [comment]
It's a regression. Broken by 2134a701c85f1a3a2fc845d65bc4158c3460db37

**Comment by Anty-Lemon** [comment]
Found the [offending line of code](https://github.com/mgba-emu/mgba/blob/medusa/src/gba/renderers/software-bg.c#L24).

Relevant symbol data:
```
renderer->d.vramBG = {0x0 <repeats 32 times>}
screenBase = 5824
```

**Comment by yuriks** [comment]
This still happens as of eec3723 (I haven't verified that the stacktrace is the same however.)

Starting the game in melonDS and then saving allows the saved game to be loaded in medusa and the game then seems to function normally so far. (It even passes the anti-piracy check! :P)

**Comment by endrift** [comment]
Fixed in de55e441520f1678864c7fe9d1936a013b096a65.
